edx_event_bus_kafka/internal/consumer.py

```diff
@@ -274,6 +286,25 @@ def reset_offsets(consumer, partitions):
         finally:
             self.consumer.close()
 
+    def consume_indefinitely(self, offset_timestamp=None):
+        """
+        Consume events from a topic in an infinite loop.
+
+        Or, if timestamp supplied, call ``reset_offsets_indefinitely`` instead and do not consume.
+        This option is deprecated, and that method should be called directly instead.
```

Comment:
It's not entirely clear what "This" refers to in the second sentence. Maybe something like:
```
Arguments:
    offset_timestamp (type?): (deprecated, optional) If timestamp supplied, call
        ``reset_offsets_indefinitely`` instead and do not consume.  This option is
        deprecated, and that method should be called directly instead.
```

Reply:
Adjusted a bit.